daemon/src/model/cfd.rs

```diff
@@ -654,8 +654,6 @@ impl Cfd {
 
     fn can_settle_collaboratively(&self) -> bool {
         !self.is_closed() && !self.commit_finality && !self.is_attested()
-            // Rollover and collaborative settlement are mutually exclusive, if we are currently rolling over we cannot settle
```

Comment:
thanks for writing detailed reasoning for removing these lines! much appreciated!